src/components/CardSelector/index.js

```diff
@@ -19,41 +22,73 @@ CardSelector.propTypes = {
+        display: "flex",
+        flexDirection: "column",
+        overflow: "unset",
+      }}
       component={Paper}
     >
       <CardHeader className={classes.cardHeader} title={title} />
-      <Collapse in={true} className={classes.collapse}>
-        <List className={classes.list} dense component="div" role="list">
-          {list.map((value, index) => {
-            return (
-              <ListItem
-                key={value + index}
-                role="listitem"
-                button
-                onClick={() => {
-                  onInputChange(value, title);
-                }}
-              >
-                <ListItemIcon>
-                  <Checkbox
-                    style={{ color: "black" }}
-                    id={value[0]}
-                    checked={value[1]}
-                    tabIndex={-1}
-                    disableRipple
+      <List className={classes.list} dense component="div">
+        {title === PERSON_LABELS.PERSONAS
+          ? list.map(([p, value]) => {
+              return (
+                <ListItem
+                  key={p.id}
+                  role="listitem"
+                  button
+                  onClick={() => {
+                    onInputChange([p, value], title);
+                  }}
+                >
+                  <ListItemIcon>
+                    <Checkbox
+                      style={{ color: "black" }}
+                      id={p.id}
+                      checked={value}
+                      tabIndex={-1}
+                      disableRipple
+                      icon={<RadioButtonUncheckedIcon />}
+                      checkedIcon={<CheckCircleIcon />}
+                    />
+                  </ListItemIcon>
+                  <ListItemText
+                    className={classes.fw400}
+                    primary={p.full_name}
                   />
-                </ListItemIcon>
-                <ListItemText primary={value[0]} />
-              </ListItem>
-            );
-          })}
-          <ListItem />
-        </List>
-      </Collapse>
+                </ListItem>
+              );
+            })
+          : list.map((value, index) => {
+              return (
+                <ListItem
+                  key={value + index}
+                  role="listitem"
+                  button
+                  onClick={() => {
+                    onInputChange(value, title);
+                  }}
+                >
+                  <ListItemIcon>
+                    <Checkbox
+                      style={{ color: "black" }}
+                      id={value[0]}
+                      checked={value[1]}
+                      tabIndex={-1}
+                      disableRipple
+                      icon={<RadioButtonUncheckedIcon />}
+                      checkedIcon={<CheckCircleIcon />}
+                    />
+                  </ListItemIcon>
+                  <ListItemText primary={value[0]} />
+                </ListItem>
```

Comment:
This is kind of duplicated? I think it's a good idea creating a sub-component with this. Right?